api/unstable/crl.h

```diff
@@ -225,11 +225,6 @@ S2N_API int s2n_config_set_cert_validation_cb(struct s2n_config *config, s2n_cer
 /**
  * Indicates that the validation performed in the cert validation callback was successful.
  *
- * `s2n_cert_validation_accept()` should be called from within the cert validation callback to allow `s2n_negotiate()`
- * to continue the handshake.
- *
- * This function must not be called outside of the cert validation callback.
```

Comment:
To confirm: we're removing these comments because the C callbacks are now async? Should we replace them with some info about calling sync vs async, or is that documented in the usage guide somewhere?